static/js/constants.js

```diff
@@ -143,7 +142,6 @@ export const ELASTICSEARCH_RESPONSE = {
 export const USER_PROFILE_RESPONSE = {
   "username": SETTINGS.username,
   "filled_out": true,
-  "agreed_to_terms_of_service": true,
```

Comment:
this is still included on profiles when we get a response from the API, is there a particular reason we want to delete this here?